web/src/pages/TracesPage/QueryEditor/TraceQLEditor.tsx

```diff
@@ -61,7 +61,13 @@ export function TraceQLEditor({ tempo, query, setQuery, runQuery }: TraceQLEdito
   return (
     <CodeMirror
       id="traceql-input"
-      className="pf-c-form-control"
+      className="pf-v6-c-form-control"
+      style={{
+        paddingBlockStart: 'var(--pf-v6-c-form-control--PaddingBlockStart)',
```

Comment:
We probably should use the [tokens](https://www.patternfly.org/tokens/all-patternfly-tokens) css variables as they will make future migrations easier.

Reply:
Thanks for the info, I wasn't aware of the tokens.
I replaced all CSS variables with PF6 token variables now.